packages/notification-center/src/store/notifications-provider.context.tsx

```diff
@@ -133,6 +140,7 @@ export function NotificationsProvider({
       markAllNotificationsAsRead,
       markAllNotificationsAsSeen,
```

Comment:
Also, I feel the names for `markAllNotificationAsRead` and `markAllNotificationsAsSeen` are incorrect.
It should be appropriate to rename these to `markFetchedNotificationsAs**` according to what they do.
And then we can rename the `markAllNotificationsByFeed` to just `markAllNotifications`.
But renaming these will be a breaking change but much needed one.